todo

```diff
@@ -0,0 +1,11 @@
+1) Move EFS to using fdIndex and an internal file descriptor index
```

Comment:
These things should either:

1. Turned into issues
2. Put into the README.md (at the end after a hr line)

Reply:
I opted for option 2, they are in the README.